packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -242,7 +244,40 @@ augmentFieldsDict(Posts, {
         return getDialogueMessageContents(post, dialogueMessageId)
       }
     }
-  }
+  },
+  annualReviewMarketProbability: {
+    resolveAs: {
+      type: 'Float',
+      resolver: async (post: DbPost, args: void, context: ResolverContext) => {
+        const market = await getWithCustomLoader(context, 'manifoldMarket', post._id, async ids =>
+          Promise.all(ids.map(id => Posts.findOne({_id: id}).then(p => !!p ? getPostMarketInfo(p) : undefined)))
```

Comment:
Surely this should be `Posts.find({ _id: { $in: ids } }).fetch().then(posts => posts.map(getPostMarketInfo))`?  (Along with the other two fields doing this)